utils/src/shutdown.rs

```diff
@@ -34,6 +34,8 @@ pub struct TriggerInner {
     mpmc_channel_shutdown: AtomicBool,
     /// State of sync follower daemon.
     sync_follower_daemon_shutdown: AtomicBool,
+    /// Shutdown Applied
+    leader_notified: AtomicBool,
```

Comment:
The `TriggerInner` doesn't need to be `pub`.

Reply:
Removed